member-service/src/main/java/com/hedvig/memberservice/services/trustpilot/TrustpilotReviewServiceImpl.kt

```diff
@@ -0,0 +1,37 @@
+package com.hedvig.memberservice.services.trustpilot
+
+import com.hedvig.memberservice.external.trustpilot.TrustpilotClient
+import com.hedvig.memberservice.external.trustpilot.TrustpilotReviewLinkRequestDto
+import com.hedvig.memberservice.util.logger
+import com.hedvig.resolver.LocaleResolver
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import java.util.Locale
+
+@Service
+class TrustpilotReviewServiceImpl(
+    private val trustpilotClient: TrustpilotClient,
+    @Value("\${trustpilot.businessUnitId}") val businessUnitId: String
+): TrustpilotReviewService {
+    override fun generateTrustpilotReviewInvitation(
+        memberId: Long,
+        email: String,
+        name: String,
+        locale: Locale?
+    ): TrustpilotReviewInvitation? {
+        val locale = locale ?: LocaleResolver.DEFAULT_LOCALE
+        return try {
+            val response = trustpilotClient.createReviewLink(
+                businessUnitId,
+                TrustpilotReviewLinkRequestDto(
+                    memberId.toString(), email, name, "${locale.language}-${locale.country}"
+                )
```

Comment:
I prefer having the name on there as well for extra readability
```suggestion
                TrustpilotReviewLinkRequestDto(
                    memberId.toString(), 
                    email, 
                    name, 
                    "${locale.language}-${locale.country}"
                )
```